### ISSUE TITLE
Cannot fetch data from docs.blockstack.org

### ISSUE THREAD

**Issue by aulneau** [blockstack/app.co#356]
When fetching the FAQ data on client side navigation, it errors due to a cors problem.

**Comment by hstove** [comment]
Is this on Brave? There is a recent breaking change where Brave seems to not support CORS at all.

https://forum.blockstack.org/t/network-requests-to-gaia-blocked-on-brave/6917/2

**Comment by aulneau** [comment]
Well the issue is that we're fetching data from a json file, and it works when you load the page directly, https://app-co-staging-pr-335.herokuapp.com/mining, it loads. But navigating from the home page to the mining page results on an error on localhost for cors related to the json file at docs.blockstack.org, I wasn't able to see the issue on prod, but i'm currently assuming it's the same issue.

**Comment by aulneau** [comment]
this is the json file `https://docs.blockstack.org/develop/faq-data.json`

**Comment by hstove** [comment]
Yeah, it makes sense that it works on first page load, because then the JSON is fetched from the server. Otherwise, it fails because there is no CORS header at all for that response from `docs.blockstack.org`.

@moxiegirl we really need an `Access-Control-Allow-Origin: *` header for this FAQ JSON request. Can we do that?

If not, we'll have to create a proxy method on the app.co API that fetches this JSON.

**Comment by aulneau** [comment]
yeah I was leaning towards just an api proxy. I think it's hosted with netlify, so maybe we can define the headers that way.

**Comment by hstove** [comment]
API proxy would probably be easier for me to spin up than messing with the netlify headers.
